assets/big-image-itcr.html

```diff
@@ -2,11 +2,12 @@
   <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
   <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
   <!--<script src="assets/hideOutput.js"></script>-->
-  <link href="assets/style.css" rel="stylesheet">
+  <link href="assets/style_ITN.css" rel="stylesheet">
 </head>
         
 
 
 <div class="hero-image-container"> 
-  <img class= "hero-image" src="https://raw.githubusercontent.com/jhudsl/ITCR_Cancer_Research_Leadership/main/images/main_image.png">
+  <img class= "hero-image" src="assets/itcr_main_image.png">
+ <!-- <img class= "hero-image" src="https://raw.githubusercontent.com/jhudsl/ITCR_Cancer_Research_Leadership/main/images/main_image_dasl.png"> -->
```

Comment:
Also here

Reply:
I added the itcr_main_image.png to the template and the hero image html now uses that instead of using the one from my repo.  
or are you referring to the comment?

Reply:
The comment.

Reply:
comment is gone